scripts/generate_apply_report.py

```diff
@@ -0,0 +1,117 @@
+import argparse
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description="Apply patch report generator")
+    parser.add_argument(
+        "-patch",
+        "--patch-name",
+        required=True,
+        metavar="<string>",
+        type=str,
+        help="Patch name",
+    )
+    parser.add_argument(
+        "-bhash",
+        "--base-hash",
+        metavar="<string>",
+        default="",
+        type=str,
+        help="Baseline hash",
+    )
+    parser.add_argument(
+        "-thash",
+        "--tree-hash",
+        metavar="<string>",
+        default="",
+        type=str,
+        help="Tip of tree hash",
+    )
+    parser.add_argument(
+        "-bstatus",
+        "--base-status",
+        required=True,
+        metavar="<string>",
+        type=str,
+        help="Baseline status",
+    )
+    parser.add_argument(
+        "-tstatus",
+        "--tree-status",
+        required=True,
+        metavar="<string>",
+        type=str,
+        help="Tip of tree status",
+    )
+    parser.add_argument(
+        "-o",
+        "--output-markdown",
+        default="./issue.md",
+        metavar="<filename>",
+        type=str,
+        help="Output file name",
+    )
+    return parser.parse_args()
+
+def build_status(bhash:str, thash: str, bstatus: str, tstatus: str):
+    result = "## Apply Status\n"
+    result += "|Target|Status|\n"
+    result += "|---|---|\n"
+    result += f"|Baseline hash: {bhash}|{bstatus}|\n"
```

Comment:
Turn the hash into a link using `https://github.com/gcc-mirror/gcc/commit/{bhash}`